components/supervisor/pkg/supervisor/tasks.go

```diff
@@ -498,14 +498,14 @@ func (tm *tasksManager) watch(task *task, term *terminal.Term) {
 		duration := ""
 		if elapsed >= 1*time.Minute {
 			elapsedInMinutes := strconv.Itoa(int(math.Round(elapsed.Minutes())))
-			duration = "⏱️ Well done on saving " + elapsedInMinutes + " minute"
+			duration = "⏱️\bWell done on saving " + elapsedInMinutes + " minute"
```

Comment:
**question**: Does `\b` work? I couldn't test this with a prebuild in the preview environment. 🤷

Reply:
maybe an xterm issue :thinking:

Reply:
Hey @jeanp413! I highlighted this because I could not test it in the preview environment. Do you think it's safe to merge this as is and see how it renders in production?